apps/experiments/tables.py

```diff
@@ -157,6 +157,21 @@ class Meta:
         empty_text = "No sessions yet!"
 
 
+class ExperimentVersionsTable(tables.Table):
+    version_number = columns.Column(verbose_name="Version Number", accessor="version_number")
```

Comment:
a date would also be good

Reply:
and show if it's the default one or not?